documentation/endpoints/activity.md

```diff
@@ -52,39 +49,49 @@ Example Response:
           {
             "id": 340,
             "signup_id": 15,
-            "northstar_id": "5571df46a59db12346dsb456d",
+            "northstar_id": "1234",
             "media": {
               "url": "https://s3.amazonaws.com/ds-rogue-prod/uploads/reportback-items/18-1487182498.jpeg",
               "caption": "Captioning captions",
             },
+            "tagged": [
+              "Good Photo",
+              "Good Quote"
+            ],
+            "reactions": [
+              {
+                  "id": 31,
+                  "northstar_id": "1234",
+                  "post_id": 1,
```

Comment:
Would this `post_id` match the `id` above (in this case `340`) in a real response?